CMakeLists.txt

```diff
@@ -63,6 +68,17 @@ if( VVDEC_ENABLE_X86_SIMD )
     set_if_compiler_supports_flag( FLAG_mfloat_abi_hard -mfloat-abi=hard  )
     add_compile_options( ${FLAG_mfpu_neon} ${FLAG_mfloat_abi_hard} )
   endif()
+
+  if( VVDEC_TARGET_ARCH STREQUAL "LOONGARCH64" )
+    if( VVDEC_ENABLE_LOONGARCH64_LSX_SIMD )
+      set_if_compiler_supports_flag( FLAG_mlsx -mlsx )
+      set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mlsx")
```

Comment:
Did you mean to use the variable `FLAG_mlsx`? As in:

```CMake
      set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${FLAG_mlsx}")
      set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${FLAG_mlsx}")
```

Reply:
Other than that looks good to me. @K-os what do you think?

Reply:
Excuse me. I made a mistake here. It should be ${FLAG_mlsx}. @adamjw24